src/guides/v2.2/rest/tutorials/orders/order-add-items.md

```diff
@@ -421,6 +421,47 @@ For this example, we'll configure the Sprite Yoga Companion Kit as follows:
 
 {% endcollapsible %}
 
+This calls are performed on behalf of a guest-customer, and there is no need of token.
+
+### Add a simple product to a cart {#add-simple}
```

Comment:
I believe we need to clarify here that we are talking about a guest user. Otherwise, we have two identical topics (and identical anchors, this point needs to be adjusted as well).

Reply:
Updated as per your comments